.github/workflows/go-mod-auto-bump.yml

```diff
@@ -0,0 +1,45 @@
+# This workflow is used in order to automatically update go.mod in case one of:
+# * osmoutils
+# * osmomath
+# * ibc-hooks
+# is updated when a push to the main branch happened (including merged PRs)
```

Comment:
Not only `main` anymore, right? Can you please update this if so?

Reply:
yeah, any branch now, gonna update right away